safe/gui/tools/help/definitions_help.py

```diff
@@ -1072,7 +1072,7 @@ def definition_to_message(
 
             if inasafe_class.get('fatality_rate') >= 0:
                 rate = inasafe_class['fatality_rate'] * 100
-                rate = u'%s%%' % rate
+                rate = u'%s%%' % format(rate, 'f')
```

Comment:
Just a suggestion, perhaps you want to change it to:

```
'{:.0%}'.format(rate)
```

To make it similar with other formatting style.

P.S.: Rate doesn't have to be multiplied by 100, in this case. So, rate = 0.5 will yield 50%.